src/index.html

```diff
@@ -77,13 +77,23 @@
                 name="routerName"
                 placeholder="{{'basicRouterSetup.routerName.placeholder' | translate }}"
                 data-ng-model="wizard.router.name"
+                data-ng-required="(wizard.internet.share && wizard.internet.vpn03.enabled && state.internet.vpn03.generate) || state.ip.register"
                 required pattern="[A-Za-z0-9_\-]{0,16}"
                 >
               <span class="fa form-control-feedback"
                 ng-class="formControlFeedback('routerName')"></span>
-              <p class="text-muted" translate>
+              <p class="text-muted" ng-hide="hasError('routerName')" translate>
```

Comment:
In the current firmware the name gets generated from the IPs if the field is empty. there was also a TODO in the sources for generating the router name. but we can not generate if ips are registrated during configuration. that's why I added this.
But: I'm also ok with making this an required field in any constellation